completers/common/fnm_completer/cmd/listRemote.go

```diff
@@ -0,0 +1,27 @@
+package cmd
+
+import (
+	"github.com/carapace-sh/carapace"
+	"github.com/spf13/cobra"
+)
+
+var listRemoteCmd = &cobra.Command{
```

Comment:
Renamed the file for consistency.
I'm using `_` to signal subcommands. When it contains `-`, which does not fit variable names, I use camelCase as done here in the code.

fyi: there are some edge cases like `.._windows.go` which is reserved by go and need to be handled a bit differently, but for the most part this works fine.